docs/storage.md

```diff
@@ -129,3 +129,27 @@ Note that on the read path, Prometheus only fetches raw series data for a set of
 ### Existing integrations
 
 To learn more about existing integrations with remote storage systems, see the [Integrations documentation](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage).
+
+## Backfilling from OpenMetrics format
+
+### Overview
+
+If a user wants to create blocks into the TSDB from data which is in [OpenMetrics](https://openmetrics.io/) format, they can do so using backfilling. However, they should be careful and note that it is not safe to backfill data from the last 3 hours (the current head block).
+
+
+The backfill sub-command is implemented such that it keeps at most 2 hours of data in the memory. Hence, it is important to make sure that there is enough RAM for 2 hours worth of data at a time.
+
+An example use case of backfilling is :
+If a user wants to migrate from an existing monitoring system (source) to Prometheus, they can add their older data to TSDB easily using backfilling. The source system may be another TSDB which is capable of dumping data in [OpenMetrics](https://openmetrics.io/) format. 
+
+### Usage 
```

Comment:
We've no mention here of what you do with the blocks once they're generated, which is not something that is likely going to be obvious to users.

So what I'd suggest is after the invocation mention moving the blocks, and mention the two gotchas in that context.